### PR TITLE
Add timeout event to shutdown HTTP conf server

### PR DESCRIPTION
**Pull request by grydz**
This PR adds:

- Timeout event to shutdown the HTTP configuration server after 180 seconds
- Debug mode for `Certificate` class without RA-TLS
- Unit tests for the HTTP configuration server
- Fix to restart without configuration server

### CODE REVIEW

File: src/mse_lib_sgx/cli.py [hunk 1]
```diff
@@ -170,6 +159,7 @@ def run() -> None:
         certificate=cert,
         uuid=args.uuid,
         need_ssl_private_key=ssl_app_mode == SslAppMode.CUSTOM_CERTIFICATE,
+        timeout=180,
```
Comment (heavenboy8) [inline src/mse_lib_sgx/cli.py]: Set `180` in a global?

File: tests/test_cert_utils.py [hunk 2]
```diff
@@ -5,8 +5,15 @@ def test_wildcard():
...
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.cosmian.app")
+        == "*.cosmian.app"
+    )
+    assert (
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.dev.cosmian.app")
+        == "*.dev.cosmian.app"
+    )
+    assert (
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.stagging.cosmian.app")
+        == "*.stagging.cosmian.app"
```
Comment (heavenboy8) [inline tests/test_cert_utils.py]: `staging`

File: src/mse_lib_sgx/cli.py (hunk 1 above)
Comment (grydz) [reply src/mse_lib_sgx/cli.py]: Fixed in f438dd769553cbb6251098fce161bf95177ba429.

File: tests/test_cert_utils.py (hunk 2 above)
Comment (grydz) [reply tests/test_cert_utils.py]: Fixed in f438dd769553cbb6251098fce161bf95177ba429.

**Comment by heavenboy8** [comment]
ok to merge!

**Review by heavenboy8 — APPROVED** [review]

**Pull request merged**